### DISCUSSION TITLE
How should I properly release the connection

### DISCUSSION THREAD

**Discussion by a1292999652** [pion/webrtc#1780] [Q&A] [answered]
### Your environment.
  - Version: v3
  - Browser: Google Chrome 88

### What did you do?
I refer to the official example：
[broadcast](https://github.com/pion/webrtc/tree/master/examples/broadcast)
I modified the sample program and incorporated it into my own program
It seems everything is normal, but I should need to deal with the end of the work
I made the following modification, but I am not sure that it is correct
![微信截图_20210421185236](https://user-images.githubusercontent.com/31630317/115542974-720a2a00-a2d3-11eb-98ab-5d7362d6a573.png)
![微信截图_20210421190054](https://user-images.githubusercontent.com/31630317/115543504-ffe61500-a2d3-11eb-97b4-ef990efae7e4.png)
![微信图片_20210421185623](https://user-images.githubusercontent.com/31630317/115542984-759db100-a2d3-11eb-80a1-50dd1267143f.png)

### What did you expect?
Correctly close the connection and release resources

### What happened?
Everything looks calm, but I feel like I’m missing something
I hope the official staff will answer for me
thank you very much

**Comment by Sean-Der** [comment, marked as answer]
Hi @a1292999652, thanks for using Pion!

When you call `PeerConnection.Close()` everything is closed/resources are released. You don't need to call Close on anything else.

**Reply by a1292999652** [reply]
alright, I understand
Thank you very much for the work of your team
Thank you very much for answering me
